tests/non-js-resources/subtests/css/subtests/source/parcel/index.md

```diff
@@ -1,8 +1,12 @@
 ---
 result: partial
-issue: https://github.com/parcel-bundler/parcel/issues/4320
+issue:
+  - url: https://github.com/parcel-bundler/parcel/issues/4320
+    fixedSince: 2.0.0-beta.1
+    status: closed
+  - url: https://github.com/parcel-bundler/parcel/issues/4855
 ---
 
-Parcel supports importing CSS stylesheets as Strings using `@parcel/transformer-inline-string`. The resulting CSS is minified and inlined into the JavaScript bundle.
+Parcel supports this out-of-the box via an [import with a special scheme](<https://v2.parceljs.org/configuration/plugin-configuration/#predefined-(offical)-named-pipelines>), `bundle-text:`.
 
-The test gets a partial pass as `@parcel/transformer-inline-string` is currently completely undocumented.
+However, this creates inline sourcemaps in your code, which will inflate the size of your JavaScript in production. The only way to avoid this is to disable sourcemaps altogether.
```

Comment:
```suggestion
However, the inlined CSS string includes a full sourcemap, inflating the size of your JavaScript in production. The only way to avoid this is to disable sourcemaps altogether.
```